pkg/gadgets/run/tracer/run.go

```diff
@@ -104,6 +105,12 @@ func (g *GadgetDesc) ParamDescs() params.ParamDescs {
 			},
 			TypeHint: params.TypeString,
 		},
+		{
+			Key:         pullSecret,
+			Title:       "Pull secret",
+			Description: "Secret to use when pulling the gadget image",
+			TypeHint:    params.TypeString,
+		},
```

Comment:
I just realized this is also shown on the `ig` case, and it'll will cause ig trying to reach the api server. I think we need to fix that as in that case using a kubernetes secret doesn't make sense.